.github/config.yml

```diff
@@ -15,14 +15,17 @@ newPRWelcomeComment: >
 # Configuration for first-pr-merge - https://github.com/behaviorbot/first-pr-merge
 # Comment to be posted to on pull requests merged by a first time user
 firstPRMergeComment: >
-   &nbsp; &#9; &nbsp; &#9;  &nbsp; &#9;  &nbsp; &#9; Thank you for contributing to the Layer5 community! :tada:
-  \
-  \
-  ![Congrats!](https://github.com/layer5io/layer5/blob/master/.github/welcome/Layer5-celebration.png)
-  \
-  \
-  &nbsp; &#9; &nbsp; &#9;  &nbsp; &#9;  &nbsp; &#9; :star: Please leave a [star on the project](../stargazers). :smile:
+  Thank you for contributing to the Layer5 community! :tada:
+  
```

Comment:
As in meshery.io repo we have "Thanks for your contribution to the Layer5 **and Meshery**  community!"
so here we can have same here or maybe only Meshery
For reference: https://github.com/meshery/meshery.io/blob/master/.github/config.yml